### DISCUSSION TITLE
Minimum requirements section on Wiki

### DISCUSSION THREAD

**Discussion by gdcmarinho** [dani-garcia/vaultwarden#5075] [Ideas]
I was trying to check out if there are some docs with the minimum requirements to install Vaultwarden on my server, but I didn't find anything. I would love to see if there are any minimum requirements to install Vaultwarden.

**Comment by BlackDex** [comment]
The minimum requirements are, a cpu, memory., storage and network. And there have been reports of people running it on RPi 1 and Zero. So, not much should be needed in regards to memory as the RPi1 has 512MB of memory. But 1GB is probably better also when your database is growing.
